Automattic/wp-calypso
# Plans, Jetpack Cloud: Add New Year 2021 sale banner to connect and plans pages

#### Changes proposed in this Pull Request

* Add the "NEWPACK" 2021 sale banner to the page header in the Connect flow (both the "store" and the "plans" pages).
* Add the same banner to the header of the Plans page for Jetpack sites.
* Move `NewYear2021SaleBanner` up a level into `client/components/jetpack`, since it's now shared among several areas.

#### Testing instructions

_See also: #48545_

* Ensure all requirements from #48545 are still met.
* Verify that the sale banner is displayed correctly (with respect to time, URL params, page positioning, etc.) when you go through the Jetpack Connect flow for a self-hosted site.
* Verify that the sale banner is displayed correctly when you visit the Plans page for a Jetpack site.
* Verify that the sale banner does _not_ show on these pages for non-Jetpack sites.
* Verify that the sale banner never shows up twice in a row on the same page.

#### Reference screenshots

##### Jetpack Connect

<img width="1090" alt="image" src="https://user-images.githubusercontent.com/670067/103581267-4e94ae00-4ea1-11eb-9e11-36c982963457.png">

##### Calypso Blue

<img width="1085" alt="image" src="https://user-images.githubusercontent.com/670067/103581302-5bb19d00-4ea1-11eb-9401-35d2917855f8.png">

##### Calypso Green

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/670067/103581215-3755c080-4ea1-11eb-871a-24c4c32221c0.png">
